Frontend/ui-common/src/Engineering/Helpers/JBridge.js

```diff
@@ -1494,11 +1481,12 @@ export const showKeyboard = function(id){
     JBridge.showKeyboard(id); // imeOptions is set to IME_ACTION_SEARCH and IME_ACTION_DONE
 }
 
-export const locateOnMap = function (str, lat, lon, geoJson, coodinates) {
-  try {
-    return JBridge.locateOnMap(str, lat, lon, geoJson, JSON.stringify(coodinates));
-  } catch (err) {
-    return JBridge.locateOnMap(str, lat, lon);
+export const locateOnMap = (configObj) => {
+  try{
+      console.log(JSON.stringify(configObj))
+      return JBridge.locateOnMap(JSON.stringify(configObj));
+  } catch (err){
+      return JBridge.locateOnMap(configObj.goToCurrentLocation, configObj.lat, configObj.lon, configObj.geoJson, JSON.stringify(configObj.points));
```

Comment:
`return JBridge.locateOnMap(str, lat, lon);` this case is not being handled in this function. Make the changes accordingly.